src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,81 @@
+// eslint-disable-next-line object-curly-newline
+import { FC, useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  deleteTodo: (id: number) => void;
```

Comment:
Create type from `(id: number) => void;`

Reply:
You use (id: number) => void; a few times. The better way is to move this to some `type` and use it. It helps you optimize your code.